custodia.spec

```diff
@@ -169,6 +169,7 @@ mkdir -p %{buildroot}/%{_unitdir}
 mkdir -p %{buildroot}/%{_tmpfilesdir}
 mkdir -p %{buildroot}/%{_localstatedir}/lib/custodia
 mkdir -p %{buildroot}/%{_localstatedir}/log/custodia
+mkdir -p %{buildroot}/%{_localstatedir}/run/custodia
```

Comment:
This is not sufficient given the run dir is killed at every reboot, we need to add a tmpfiles.d snippet so that a new dir is created by systemd at startup